### PR TITLE
script to identify required releases

### PR DESCRIPTION
**Pull request by brettmc**
A script to identify which of our git-split repos have changes which are not in a release. It prints out
information about the last release version/date, and provides a link to create a new release via the
github UI.
Lastly, it generates release notes by linking commits back to the PR from upstream.

### CODE REVIEW

**Comment by pdelewski** [comment]
Have not investigate that deeply, however I'm receiving following errors 
```
ERROR: proto/otel: Latest release not found for opentelemetry-php/gen-otlp-protobuf: rate limit exceeded
ERROR: src/Context: Latest release not found for opentelemetry-php/context: rate limit exceeded
ERROR: src/SemConv: Latest release not found for opentelemetry-php/sem-conv: rate limit exceeded
ERROR: src/API: Latest release not found for opentelemetry-php/api: rate limit exceeded
ERROR: src/SDK: Latest release not found for opentelemetry-php/sdk: rate limit exceeded
ERROR: src/Contrib: Latest release not found for opentelemetry-php/sdk-contrib: rate limit exceeded
ERROR: src/Contrib/Newrelic: Latest release not found for opentelemetry-php/exporter-newrelic: rate limit exceeded
ERROR: src/Contrib/Otlp: Latest release not found for opentelemetry-php/exporter-otlp: rate limit exceeded
ERROR: src/Contrib/Grpc: Latest release not found for opentelemetry-php/transport-grpc: rate limit exceeded
ERROR: src/Contrib/Zipkin: Latest release not found for opentelemetry-php/exporter-zipkin: rate limit exceeded
ERROR: src/Contrib/ZipkinToNewrelic: Latest release not found for opentelemetry-php/exporter-zipkintonewrelic: rate limit exceeded
ERROR: src/Extension/Propagator/B3: Latest release not found for opentelemetry-php/extension-propagator-b3: rate limit exceeded
ERROR: src/Aws: Latest release not found for opentelemetry-php/contrib-aws: rate limit exceeded
ERROR: src/Symfony: Latest release not found for opentelemetry-php/contrib-sdk-bundle: rate limit exceeded
ERROR: src/Instrumentation/IO: Latest release not found for opentelemetry-php/contrib-auto-io: rate limit exceeded
ERROR: src/Instrumentation/Laravel: Latest release not found for opentelemetry-php/contrib-auto-laravel: rate limit exceeded
ERROR: src/Instrumentation/PDO: Latest release not found for opentelemetry-php/contrib-auto-pdo: rate limit exceeded
ERROR: src/Instrumentation/Psr15: Latest release not found for opentelemetry-php/contrib-auto-psr15: rate limit exceeded
ERROR: src/Instrumentation/Psr18: Latest release not found for opentelemetry-php/contrib-auto-psr18: rate limit exceeded
ERROR: src/Instrumentation/Slim: Latest release not found for opentelemetry-php/contrib-auto-slim: rate limit exceeded
ERROR: src/Instrumentation/Symfony: Latest release not found for opentelemetry-php/contrib-auto-symfony: rate limit exceeded
ERROR: src/Instrumentation/Wordpress: Latest release not found for opentelemetry-php/contrib-auto-wordpress: rate limit exceeded
ERROR: src/Context/Swoole: Latest release not found for opentelemetry-php/context-swoole: rate limit exceeded
```
Seems that releases exists for this packages.

**Comment by brettmc** [comment]
@pdelewski you're getting rate-limited by github.
You can use a token, which is documented in the script. `GH_TOKEN=<token> php create_releases.php`

**Comment by pdelewski** [comment]
> https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"

ok, thanks. Missed that usage comment

**Comment by pdelewski** [comment]
> @pdelewski you're getting rate-limited by github. You can use a token, which is documented in the script. `GH_TOKEN=<token> php create_releases.php`

ok, thanks. Missed that usage comment.

File: script/create_releases.php [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+    public function __construct()
+    {
+        self::$debug = getenv('DEBUG') !== false;
+        $token = getenv('GH_TOKEN');
+        $headers = [];
+        if ($token) {
+            $headers['Authorization'] = "token {$token}";
+        } else {
+            echo "[WARNING] No github token provided, you may get rate-limited." . PHP_EOL;
```
Comment (pdelewski) [inline script/create_releases.php:39]: I was thinking about that. thx

**Comment by pdelewski** [comment]
@brettmc GH_TOKEN=<token> php create_releases.php works as expected

**Review by pdelewski — APPROVED** [review]

**Pull request merged**